local.codecept.json

```diff
@@ -66,6 +66,12 @@
       "browsers": ["chrome"]
     }
   },
+  "plugins": {
+    "allure": {},
```

Comment:
This would require us to also install allure  on builds, why do we need allure with autodelay plugin?

Reply:
removed it